docs/walkthrough/create-training-data.md

```diff
@@ -92,6 +92,16 @@ During fine-tuning, we're optimizing two models in parallel.
 At the model saving time, you will discover, we are saving two models to your local directory. 
 ```
 
+### Query-Document Scores
+
+To prepare data for query-document pairs which are annotated with similarity scores,
+each row must contain three columns includes two sentences and a similarity score.
+
+```markdown
+The weather is nice, The weather is beautiful, 0.9
+The weather is nice, The weather is bad, 0
+```
+
 ```{important} 
 If a text field contains commas, it breaks the CSV format since it is interpreted as spanning over multiple columns.
 In this case, please enclose the field in double quotes, such as `field1,"field, 2"`.
```

Comment:
```suggestion
```{important} 
I```suggestion
If your texts contain commas, you must enclose them in quotes, otherwise you will break the CSV format.  For example:
```
```markdown
"If we're ordering lunch, we should get pizza", "I want to order pizza for lunch", 0.8
"If you're headed out, can you take out the garbage?", "I'm going to have to take the trash out myself, aren't I?", 0.1
```